src/data/gallery/gallery.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is it necessary to have cmsGallery.json and also gallery.json?

Reply:
forgot to remove it... good point